### PR TITLE
DRY up pkg/config envelope creation

### PR DESCRIPTION
**Pull request by wlahti**
#### Type of change

- Improvement (improvement to code, performance, etc)

#### Description

Reuse newEnvelope() when creating a signed envelope. 

#### Related issues

[FAB-17741](https://jira.hyperledger.org/browse/FAB-17741)

### CODE REVIEW

File: pkg/config/config.go
Comment (stephyee) [inline pkg/config/config.go]: Did you forget to delete this line?

Comment (wlahti) [reply pkg/config/config.go]: Ah yep, didn't even notice it for a while after reading your comment. :) Removed.

File: pkg/configtx/signer.go
Comment (caod123) [inline pkg/configtx/signer.go:142]: could maybe even call this just `signedEnvelope` and just drop the `new` since that seems to be the pattern we're moving towards

Comment (caod123) [reply pkg/configtx/signer.go:142]: I'm indifferent either way. I'll let you decide which you prefer, let me know if you want to merge as is or if you want to change it

**Review by caod123 — APPROVED** [review]

File: pkg/configtx/signer.go
Comment (wlahti) [reply pkg/configtx/signer.go:142]: I say we keep it as-is for now.  We can do another pass to make sure we're consistent in our naming of functions after the current set of tasks are merged in.

**Pull request merged**
